src/robot/parsing/model/statements.py

```diff
@@ -181,6 +213,13 @@ class SectionHeader(Statement):
                      Token.TESTCASE_HEADER, Token.KEYWORD_HEADER,
                      Token.COMMENT_HEADER)
 
+    @classmethod
+    def from_params(cls, type, name, separator=FOUR_SPACES, eol=EOL):
+        return cls([
+            Token(type, name),
+            Token('EOL', '\n')
+        ])
```

Comment:
`separator` isn't used for anything. Is it in the signature for consistency with other `from_param` methods or by accident? I don't think consistency is a good enough reason to have it there and it should be removed.

Reply:
It's by accident, thanks for noticing